RCSID $Id: //WIFI_SOC/MP/SDK_4_3_0_0/RT288x_SDK/source/user/openswan-2.6.22/linux/net/ipv4/af_inet.c.fs2_4.patch#1 $
--- linux/net/ipv4/af_inet.c.preipsec	Wed Apr 26 15:13:17 2000
+++ linux/net/ipv4/af_inet.c	Fri Jun 30 15:01:27 2000
@@ -1019,6 +1019,17 @@
 	ip_mr_init();
 #endif
 
+#if defined(CONFIG_KLIPS)
+	{
+               extern /* void */ int ipsec_klips_init(void);
+		/*
+		 *  Initialise AF_INET ESP and AH protocol support including 
+		 *  e-routing and SA tables
+		 */
+		ipsec_klips_init();
+	}
+#endif /* CONFIG_KLIPS */
+
 	/*
 	 *	Create all the /proc entries.
 	 */
